dist/locale/multiple-select-en-US.js

```diff
@@ -2,9 +2,9 @@
   typeof exports === 'object' && typeof module !== 'undefined' ? factory(require('jquery')) :
```

Comment:
It is not necessary to modify the `dist` folder. Because when we release a new version will auto-update the `dir` folder.

Reply:
Sure, sorry. I've amended to remove.